docs/faq.md

```diff
@@ -0,0 +1,28 @@
+## FAQ
+
+### What is a bounded load-test?
+
+We define a load-test to be bounded when the number of simulated users is fixed.
+This is particularly useful when running performance comparisons between two clusters/builds.
+
+#### Note
+
+To manually run a bounded load-test using the [`coordinator`](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/coordinator.md) the [feedback loop](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/coordinator.md#the-feedback-loop) should be disabled. This can be done by either removing `MonitorConfig.Queries` or by disabling each query (by setting `Alert` to `false`).
+
+### What is an unbounded load-test?
+
+We define a load-test to be unbounded when the number of simulated users can vary up to a pre-configured limit.
+This type of load-test is used to determine the capacity of a system and will output an estimated number of users.
+The rule of thumb is that when starting an unbounded load-test we should always shoot for more users than what we think an installation can support.
+`ClusterConfig.MaxActiveUsers`  should be set to  `AgentInstanceCount * UsersConfiguration.MaxActiveUsers`.
+
+## Troubleshoot
+
+### Users are not connecting
+
+Make sure that both `Enable Account Creation` and `Enable Open Server` settings are set to `true` in MM System Console.
+
+### Agent logs show several `current team should be set` errors.
```

Comment:
Wasn't there a similar error like `no team found` in the memstore? Should we add that too?

Reply:
I think there was a bug at some point which we fixed. Now the error would be `current team should be set` if we expect the team to be there but none found.